integration/angular/simple-message.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable */
+import { util, configure, Writer, Reader } from 'protobufjs/minimal';
```

Comment:
@PhilipMantrov do you know all of these other files changed?

Reply:
Newest version of protobufjs has deep types, so compiler dont need abstraction as _m0 anymore. 
In package.json we have '^', so we install newest version 6.11.3 (on 6.8.8 how i see compiler use _m0 for abstraction).

Reply:
Huh! I'd love to get rid of this, but I'm not following why this would change, b/c:

1) Isn't yarn.lock pinned to 6.8.8?
2) What controls the `_m0` output is this line:

https://github.com/stephenh/ts-proto/blob/main/src/options.ts#L183

I.e. the `* as _m0` is generated by ts-poet because we explicitly tell it to, and I'm like 99% sure nothing in `node_modules/protobuf/...` would affect ts-poet's behavior as to whether it does/does not use the `_m0` style import...

So, dunno, I'm still not following how this changed?

Reply:
@PhilipMantrov I checked out this PR locally, and when I run `yarn bin2ts`, AFAICT it puts all of the `_m0` back... I'm wondering if maybe when you ran codegen, you weren't on the latest `main` and/or had an outdated version of `ts-poet` installed?

Fwiw I'd be really happy to bump protobufjs and clean up these imports, but seems like something to do in a separate PR.

Can you try a new `yarn bin2ts` and see what happens? Thanks!

Reply:
@PhilipMantrov I got your ping about asking for a re-review, see this comment ^.

If we just can't figure this out, I can always create a new PR with your core code changes, and w/o all of the `_m0` changes, and get that merged.

Reply:
protobufjs now fixed on 6.8.8

Reply:
>I'd be really happy to bump protobufjs and clean up these imports, but seems like something to do in a separate PR  
I create another PR for this sooner.